app/models/mixins/supports_feature_mixin.rb

```diff
@@ -73,15 +73,13 @@ module SupportsFeatureMixin
     :cinder_service             => 'Cinder storage service',
     :create_floating_ip         => 'Floating IP Creation',
     :create_host_aggregate      => 'Host Aggregate Creation',
-    :create_network_router      => 'Network Router Creation',
```

Comment:
@durandom, https://github.com/ManageIQ/manageiq-providers-openstack/pull/56 has been merged, could you please revisit?